powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/configuration/PowerAuthServiceConfiguration.java

```diff
@@ -264,6 +264,18 @@ public class PowerAuthServiceConfiguration {
     @Value("${powerauth.service.proximity-check.otp.length:8}")
     private int proximityCheckOtpLength;
 
+    /**
+     * Time-step size used for generating and validating TOTP for the proximity check.
+     */
+    @Value("${powerauth.service.proximity-check.otp.step-length:30s}")
+    private Duration proximityCheckStepLength;
```

Comment:
Let's use proximityCheckStepDuration. The step length in terms of time duration is super confusing.

Reply:
ok, it might be confusing. Renamed to step duration.